src/components/EntryEditor/EntryEditor.js

```diff
@@ -1,15 +1,19 @@
 import React, { Component, PropTypes } from 'react';
 import ImmutablePropTypes from 'react-immutable-proptypes';
 import SplitPane from 'react-split-pane';
+import Button from 'react-toolbox/lib/button';
 import { ScrollSync, ScrollSyncPane } from '../ScrollSync';
 import ControlPane from '../ControlPanel/ControlPane';
 import PreviewPane from '../PreviewPane/PreviewPane';
 import Toolbar from './EntryEditorToolbar';
 import styles from './EntryEditor.css';
 
+const PREVIEW_STATE = 'cms.preview-state';
+
 class EntryEditor extends Component {
   state = {
     showEventBlocker: false,
+    previewOpen: localStorage.getItem(PREVIEW_STATE) === "open",
```

Comment:
We should simplify this by using a boolean - name it `PREVIEW_HIDE` and set to `'true'` on hide. It'll still be stored as a string, but it's better than checking for custom string values.